docker-compose.yml

```diff
@@ -1,27 +1,51 @@
-version: '3'
+version: "3"
 services:
   db:
     image: timescale/timescaledb-postgis:1.7.4-pg11
     environment:
-        - POSTGRES_USER=postgres
-        - POSTGRES_PASSWORD=postgres
-        - POSTGRES_DB=monitorfishdb
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=monitorfishdb
     volumes:
       - db-data:/var/lib/postgresql/data
     ports:
       - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 1s
+      timeout: 1s
+      retries: 30
 
-#  geoserver:
-#    image: kartoza/geoserver:2.18.0
-#    restart: unless-stopped
-#    ports:
-#      - 8081:8080
-#    links:
-#      - "db:db"
-#    volumes:
-#      - geoserver-data:/opt/geoserver/data_dir
-#    depends_on:
-#      - db
+  flyway:
```

Comment:
I did that because it's logical to have a root docker-compose that describe all the services that will can be run on a repository and extend from that everywhere else we need it.

For example, running the Cypress-dedicated docker-compose for local development is king of strange, at least to me.

Reply:
But let's modifiu the `run-stubbed-apis` command then to use this service